docs/UserGuide.md

```diff
@@ -363,7 +364,7 @@ will display all rooms that have been cleaned.
 
 ### Listing all guests: `list`
 
-This command displays a list of all guests in GuestBook.
+This command displays a list of all guests in GuestBook, providing you a quick and efficient way to view all your guests.
```

Comment:
To be honest, I think the list command might not be the most efficient way to view the guests? Cuz you would need to scroll a lot if you want to view all the guest? Maybe we can say like the list command is like the home page? It has no filter placed on the list? I'm not too sure, any thoughts @jehousoh?

Reply:
Either "go back to the home page" or this is fine imo.

Reply:
Will note!